src/test/kotlin/build/buf/gradle/ConfigurationTest.kt

```diff
@@ -28,8 +28,23 @@ class ConfigurationTest : AbstractBufIntegrationTest() {
         assertFailure()
     }
 
+    @Test
+    fun `project can use both buf-yaml and the protobuf-gradle-plugin, protobuf applied first`() {
+        assertSuccess()
+    }
+
+    @Test
+    fun `project can use both buf-yaml and the protobuf-gradle-plugin, protobuf applied second`() {
+        assertSuccess()
+    }
+
     private fun assertFailure() {
         val result = gradleRunner().withArguments(":tasks").buildAndFail()
         assertThat(result.output).contains("cannot use both the protobuf-gradle-plugin and a Buf workspace")
     }
+
+    private fun assertSuccess() {
+        val result = gradleRunner().withArguments(":tasks").build()
+        assertThat(result.output).doesNotContain("cannot use both the protobuf-gradle-plugin and a Buf workspace")
```

Comment:
This feels a little bit brittle. If we tweak the wording of the error message, we could then break what this test is trying to cover, but this assertion would mean the test case actually passes. Is there nothing in the output to look for to change this to a `contains` call?